packages/app-content-pages/server/server.js

```diff
@@ -2,7 +2,7 @@ if (process.env.NEWRELIC_LICENSE_KEY) {
   require('newrelic')
 }
 
-const Sentry = require('@sentry/node')
+const Sentry = require('@sentry/nextjs')
```

Comment:
`@sentry/nextjs` exports all of `@sentry/node`. 
https://github.com/getsentry/sentry-javascript/blob/700d28046c9ec752c7600e0aa0d14c5a6cedac3e/packages/nextjs/src/server/index.ts#L14